### PR TITLE
Lumen value updated

### PR DESCRIPTION
**Pull request by iliaplatone**

### CODE REVIEW

File: libs/indicom.h [hunk 1]
```diff
@@ -370,11 +370,11 @@ double m2au(double m);
...
- * @brief calc_photon_flux Returns the photon flux of the object with the given magnitude observed at a determined wavelenght using a passband filter over an incident surface
+ * @brief calc_photon_flux Returns the photon flux of the object with the given magnitude observed at a determined wavelenght using a passband filter through a steradian expressed cone
  * @param rel_magnitude Relative magnitude of the object observed
  * @param filter_bandwidth Filter bandwidth in meters
  * @param wavelength Wavelength in meters
- * @param incident_surface The incident surface in square meters
+ * @param steradian The light cone in steradians
  * @return the photon flux in Lumen
  */
 double calc_photon_flux(double rel_magnitude, double filter_bandwidth, double wavelength, double incident_surface);
```
Comment (knro) [inline libs/indicom.h:380]: last parameter needs to be renamed?

Comment (iliaplatone) [reply libs/indicom.h:380]: please wait some minutes I'm working on this right now. sorry.

Comment (iliaplatone) [reply libs/indicom.h:380]: Can you re-open the interferometer PR in 3rdparty?

Comment (iliaplatone) [reply libs/indicom.h]: Allright, this is mergeable

Comment (iliaplatone) [reply libs/indicom.h]: This one can be merged, thanksfor your patience.

**Pull request merged**